examples/list_query_usage/list_query_usage.py

```diff
@@ -22,7 +22,7 @@
     # Make API call(s) to AgentQL server to fetch the query
     response = session.query(QUERY)
 
-    socks_data = response.socks.to_data()
+    socks_data = response.to_data()
```

Comment:
same here, not sock anymore. Btw: why can't we use `.socks.to_data()` now?

Reply:
I think it's because of the following line in `to_data()` method:

if not isinstance(res, dict):
            raise TypeError("Root node must be a container node")
            
We are enforcing that the result must be of `dictionary` type. Since `socks` is of `ContainerList` node type, performing a `to_data()` on this method will return a `list` of dictionaries.

I think that was a conscious decision to enforce calling `to_data()` method on root node. @paveldudka might have more context on this. Here is the PR that made this change:

https://github.com/tinyfish-io/agentql-client/pull/362/files